### PR TITLE
Added  Air Force Academy Chapel

### PR DESCRIPTION
**Pull request by MustaphaTR**
Also adds it to the Allied-2 Map.
aircforce academy

### CODE REVIEW

**Comment by MustaphaTR** [comment]
This also needs rebase. But i don't have time now.

**Comment by Phrohdoh** [comment]
```ini
;Air Force Academy Chapel
[CACOLO01]
Name=RA2 Air Force Academy Chapel Colorado
UIName=Name:CACOLO01
TechLevel=-1
Strength=1000
Insignificant=yes
Nominal=yes
RadarInvisible=yes
Points=5
Armor=concrete
Explosion=TWLT070,S_BANG48,S_BRNL58,S_CLSN58,S_TUMU60
;DestroyAnim=CARUS01D
DebrisAnims=DBRI-WM1,DBRI-WM2,DBRI-WM3,DBRI-WM4,DBRI-WM5,DBRI-WM6
MaxDebris=15
MinDebris=5
CrateBeneath=true
CrateBeneathIsMoney=true
;Selectable=no
;IsBase=no
BaseNormal=no ;psst....IsBase isn't a Rules flag
Sight=6 ; UC base values
ClickRepairable=no
CanBeOccupied=no
;MaxNumberOccupants=10
;DistributedFire=yes
;Primary=AlliedOccupyW
;Secondary=SovietOccupyW
LeaveRubble=yes
```

http://cnc.wikia.com/wiki/Air_Force_Academy

Can someone confirm the proper ingame name for this actor?

**Comment by MustaphaTR** [comment]
I copied it from .csf editor XCC Mixer. I'm sure this is the true one.

**Comment by abcdefg30** [comment]
My game also says that `Air Force Academy Chapel` is correct.

**Comment by Phrohdoh** [comment]
Great!
@MustaphaTR Is there any way you could export all of the csf strings in XCC Mixer?

**Comment by MustaphaTR** [comment]
There is no Automatic way to do it, it should take time but i can.

**Comment by MustaphaTR** [comment]
I made a PNG list of NAME:'s. It is really long. This list possibly contains some leftover stuff from my mod when it was a vanilla RA2 mod but what we need is vaule of orginal tags so it shouldn't be a problem.

Warning huge image!

csf

**Comment by Phrohdoh** [comment]
Cool, nice job!

**Comment by MustaphaTR** [comment]
Rebased.

**Comment by Phrohdoh** [comment]
This looks sensible.
@abcdefg30 could you get a proper review in for this?

File: rules/civilian-structures.yaml [hunk 1]
```diff
@@ -358,3 +358,18 @@ caukfgl:
 	Inherits: ^Flag
 	Tooltip:
 		Name: British Flag
+
+cacolo01:
+	Inherits: ^CivBuilding
+	Selectable:
```
Comment (abcdefg30) [inline rules/civilian-structures.yaml]: You shouldn't need this trait, it is already inherited.

**Comment by MustaphaTR** [comment]
Updated.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by Mailaender** [comment]
My experimental RULES.INI and ART.INI importers may be interesting for you. They will still require manual polish, but might get the grunt work done a bit quicker.

* https://github.com/Mailaender/OpenRA/tree/import-rules
* https://github.com/Mailaender/OpenRA/tree/import-sequences

**Pull request merged**